thirdparty/patches/AwsFindPackage.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This file is there at `crt/aws-crt-cpp/crt/aws-c-common/cmake/AwsFindPackage.cmake` downloaded by `prefetch_crt_dependency.sh` script, and included in aws-sdk-cpp/CMakeFiles.txt when BUILD_DEPS is ON (the default value). Is it still necessary?

Reply:
You are right. Let me try to build without copying AwsFindPackage.cmake.